### PR TITLE
Mc cleanup

### PR DESCRIPTION
**Pull request by FBRTMaka**
@DanielJMaher @birdage , minor fixes:

1. Cleaned up the plot control buttons (got messed up a little on some css changes)
2. Fixed the event class column on the plotting page (now shows up again)
3. Added an info button next to the subscription button (per Eoin)

### CODE REVIEW

**Comment by DanielJMaher** [comment]
4 might cause us some issues.  There are requirements for Json and CSV.  But since they are going through ERDDAP at some point, we would be fine.   

I'm a little worried about doing this, but  ithink it might be fine.

**Comment by FBRTMaka** [comment]
@DanielJMaher great find that this works now for the download!  Just re-enabled it!

File: ooiui/static/js/partials/StreamDownloadForm.html [hunk 1]
```diff
@@ -10,10 +10,13 @@
         <img width="50px" src="/img/logos-banners/OOI_Logo.svg">
 
 
-          <span style="margin-left:25%">
+          <span class="subscription-selection" style="margin-left:25%">
               <button style="padding-top:5px;" id="subscription-selection-select" type="button" class="btn btn-default btn-xs">
                 <span id="subscription-selection-icon" class="fa fa-heart-o" aria-hidden="true"></span> Enable Subscription
               </button>
+              <span class="subscribe-info">
+                  <i class="fa fa-info-circle" data-title="Enabling subscriptions will keep you updated when ever a stream changes."></i>
```
Comment (DanielJMaher) [inline ooiui/static/js/partials/StreamDownloadForm.html]: Should read "Enabling subscription to a stream will cause the data provision service to auto-generate this file once a day.  The user may retrieve it on one of the OPeNDAP servers." or equivalent.

File: ooiui/static/js/partials/StreamDownloadForm.html [hunk 2]
```diff
@@ -84,7 +87,7 @@
         <div class="col-md-6">
           <div class="form-group" id="type-select">
             <label for="type-select">File Type</label>
-            <select class="form-control">
+            <select class="form-control" title="Currently downloads are only available from the data source as NetCDF format">
```
Comment (DanielJMaher) [inline ooiui/static/js/partials/StreamDownloadForm.html]: Grammar Freak.  Sorry.

"Currently, downloads are only available from the data source in NetCDF format"

File: ooiui/templates/science/streams.html [hunk 3]
```diff
@@ -48,6 +48,11 @@
                 <div id="streamPanel"class="panel">
                     <div class="panel-heading">
                         <div class="pagination pagination-sm pagination-right pagination-boxes"></div>
+                        <div class="color-index">
+                            <div class="twelve-hours"></div><b> 12 Hours </b>
+                            <div class="twenty-four-hours"></div><b> 24 Hours </b>
+                            <div class="older"></div><b> Older than 24 hours </b>
+                        </div>
```
Comment (DanielJMaher) [inline ooiui/templates/science/streams.html]: I thought you were putting a legend in here for what each color means.  Or did i miss that?

Comment (FBRTMaka) [inline ooiui/templates/science/streams.html]: @DanielJMaher totes missed it:
screen shot 2015-11-19 at 4 27 59 pm

File: ooiui/static/js/partials/StreamDownloadForm.html (hunk 2 above)
Comment (FBRTMaka) [inline ooiui/static/js/partials/StreamDownloadForm.html]: @DanielJMaher Not freakishly enough, apparently you don't care about the period at the end!!

Comment (FBRTMaka) [inline ooiui/static/js/partials/StreamDownloadForm.html]: @DanielJMaher and, also freakishly forgot that this entire sentence shouldn't even be here – we DO support the other formats! :D

File: ooiui/templates/science/streams.html (hunk 3 above)
Comment (DanielJMaher) [inline ooiui/templates/science/streams.html]: Whoops. I read this as something totally different.

File: ooiui/static/js/partials/StreamDownloadForm.html (hunk 2 above)
Comment (DanielJMaher) [inline ooiui/static/js/partials/StreamDownloadForm.html]: I was thinking of putting in a whole things about OPeNDAP only supporting the NetCDF, but i was way too lazy for that.

**Comment by FBRTMaka** [comment]
reviewed with Dan

**Pull request merged**